examples/aws-microk8s/cloud-formation.yaml

```diff
@@ -467,15 +473,26 @@ Resources:
     Type: AWS::EC2::SecurityGroup
     Properties:
       GroupDescription: Cluster communication with worker nodes
+      VpcId: !Ref VPC
       SecurityGroupIngress:
         - !If
           - ShouldAllowSSH
           - Description: "SSH"
             IpProtocol: tcp
             FromPort: 22
             ToPort: 22
-            CidrIp: 0.0.0.0/0
+            CidrIp: !Ref AllowSSHCidr
           - !Ref AWS::NoValue
+      SecurityGroupEgress:
+        - IpProtocol: -1
+          CidrIp: 0.0.0.0/0
+  ClusterCommunicationIngress:
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      GroupId: !GetAtt NodeSecurityGroup.GroupId
+      Description: "In-between node communication"
```

Comment:
I am not sure what in-between means here? You mean node-to-node and pod-to-node?

Reply:
This is to enable traffic between cluster's nodes, so node to node . I'll update the description to word it better.